packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -518,10 +518,13 @@ export default class ModelViewerElementBase extends ReactiveElement {
    * Called on initialization and when the resize observer fires.
    */
   [$updateSize]({width, height}: {width: any, height: any}) {
+    width = Math.floor(parseFloat(width));
+    height = Math.floor(parseFloat(height));
```

Comment:
Your last comment got me thinking. I believe you're right and we should change these to `width = Math.ceil(parseFloat(width) * dpr) / dpr;`

Reply:
This code is now actually left untouched (.container retaining floating point dimensions appears to be a better solution in conjunction with the new ceil() approach).